modules/installation/proc-installing-with-helm-cli.adoc

```diff
@@ -0,0 +1,56 @@
+[id="installing-with-helm-cli_{context}"]
+= Installing {product} with Helm CLI
+
+You can use the Helm CLI to install {product} on OpenShift or Kubernetes.
+
+.Prerequisites
+* An `oc` or `kubectl` session with a user that has the `developer` or `admin` role on an OpenShift or Kubernetes cluster.
+* Helm CLI.
+
+.Procedure
+. For a Kubernetes cluster, you  must first add the OpenShift Helm Charts repository. On Openshift, this is already present. 
++
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ OpenShift project or Kubernetes namespace:
++
+[subs="quotes+"]
+----
+$ NAMESPACE=_<rhdh>_
+$ oc new-project ${NAMESPACE} || oc project ${NAMESPACE}
```

Comment:
do we need these `$` in the commands? I can't recall if they exist in the rest of the docs, but https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html-single/administration_guide_for_red_hat_developer_hub/index#proc-install-rhdh-airgapped-environment-ocp-helm_admin-rhdh has no `$` prefixes so it's easier to copy and paste